Fix scripts/updating incident records and disables associated workflows  based on specific criteria using two functions /script.js

```diff
@@ -0,0 +1,23 @@
+updateWorkNotes();
+updateResolve();
+function updateWorkNotes() {
+	var gr = new GlideRecord('incident');
+	gr.addEncodedQuery('priority=1^state=2^category=software');
+	gr.query();
+	while(gr.next()) {
+		gr.setValue('work_notes', 'Auto Update');
+		gr.update();
+	}
+}
+function updateResolve() {
+	var gr1 = new GlideRecord('incident');
```

Comment:
gr1 is just as bad if not worse - please use a more descriptive and unique variable name